src/main/java/seedu/address/logic/commands/ListMemberCommand.java

```diff
@@ -8,7 +8,7 @@
 /**
  * Lists all persons in the address book to the user.
  */
-public class ListCommand extends Command {
+public class ListMemberCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
```

Comment:
```suggestion
    public static final String COMMAND_WORD = "listm";
```
I think the command should be `listm` instead of `list`.

Reply:
i agree! will edit as necessary and merge